app/src/main/java/de/danoeh/antennapod/fragment/swipeactions/RemoveFromHistorySwipeAction.java

```diff
@@ -0,0 +1,66 @@
+package de.danoeh.antennapod.fragment.swipeactions;
+
+import android.content.Context;
+
+import androidx.fragment.app.Fragment;
+
+import com.google.android.material.snackbar.Snackbar;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.MainActivity;
+import de.danoeh.antennapod.core.storage.DBWriter;
+import de.danoeh.antennapod.model.feed.FeedItem;
+import de.danoeh.antennapod.model.feed.FeedItemFilter;
+
+public class RemoveFromHistorySwipeAction implements SwipeAction {
+
+    @Override
+    public String getId() {
+        return REMOVE_FROM_HISTORY;
+    }
+
+    @Override
+    public int getActionIcon() {
+        return R.drawable.ic_delete;
```

Comment:
Hmm, please use a unique icon here. Maybe somehow combining the history icon with a little X, like we do on the queue screen?